### PR TITLE
Docs 860 part2: persistant queues

### PR DESCRIPTION
**Pull request by feorlen**
Docs updates for persistent queues added in https://github.com/minio/minio/pull/17121

Kafka bit needs additional attention, see https://github.com/minio/docs/issues/886

Staged:
http://192.241.195.202:9000/staging/DOCS-860-part2/linux/html/reference/minio-server/minio-server.html#logging
http://192.241.195.202:9000/staging/DOCS-860-part2/linux/html/reference/minio-mc-admin/mc-admin-config.html#http-webhook-log-target

Partly addresses https://github.com/minio/docs/issues/860

### CODE REVIEW

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-config.rst:213]: This is a guess: does `queue_dir` require setting `queue_size`?

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-config.rst:298]: I don't see a `MINIO_AUDIT_WEBHOOK_QUEUE_SIZE`, I'm confused that you can set a path but not the size of the queue to write there.

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-config.rst:310]: I made up this name because I don't know what the corresponding command is for `MINIO_AUDIT_KAFKA_QUEUE_DIR`

Answer TBD

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-config.rst:322]: I made up this name because I don't know what the corresponding command is for `MINIO_AUDIT_KAFKA_QUEUE_SIZE`

Answer TBD

**Comment by feorlen** [comment]
@Praveenrajmani I have several questions about how the new settings/envvars work (see comments above). I made some guesses as placeholders for the correct information.

I expected each of the new envvars would have an `mc` equivalent, but what are those settings for Kafka? For example, if `MINIO_AUDIT_WEBHOOK_QUEUE_DIR` corresponds to `queue_dir` then what is `MINIO_AUDIT_KAFKA_QUEUE_DIR`?

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (ravindk89) [inline source/reference/minio-mc-admin/mc-admin-config.rst:199]: I wonder if a user would read 'such as...' as being related to the message itself. Maybe

```suggestion
Specify the directory path, such as ``/opt/minio/events``, to enable MinIO's persistent event store for undelivered messages.
```

Also suggest ``/opt/minio`` or something similar. ``/home`` might not work since we usually run as a system service.

We might need to add a line "The MinIO process must have read/write/list access on the specified directory" since not everyone thinks about permissions.

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-config.rst:213]: Since the default is ``100000`` my guess is *no*, we'll just use the default.

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-config.rst:298]: https://github.com/minio/minio/blob/master/internal/logger/config.go#L73-L79

Looks like it's in the code, but we missed it in the actual docs - which...happens. You can add it if you want in this run.

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-config.rst:310]:
```
mc admin config get play audit_kafka                                                                       ✔  docs   system   
audit_kafka enable=off topic= brokers= sasl_username= sasl_password= sasl_mechanism=plain client_tls_cert= client_tls_key= tls_client_auth=0 sasl=off tls=off tls_skip_verify=off version= queue_size=100000 queue_dir=
```

There are ways to find this in code too, but this is far easier. 

So this actually needs the root `.. mc-conf:: audit_kafka` key first.

I think I didn't get around to documenting the kafka stuff - we can drop that for now and create a new issue for kafka audit, which I'll take responsibility for - just to unblock this.

**Review by ravindk89 — CHANGES_REQUESTED** [review]
Minor nits and a "Oh I did that" for Kafka stuff.

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-config.rst:199]: If `home/events` is a bad choice, should the other instances of that get updated also? I got that text from a similar queue path setting.

Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-config.rst]: Added your reword manually 👍🏻

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-config.rst]: Here's the new issue
https://github.com/minio/docs/issues/886

Let's discuss next week. If I sufficiently understand what is needed, no reason for you to take this on.

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-config.rst]: On hold, see https://github.com/minio/docs/issues/886

File: source/reference/minio-mc-admin/mc-admin-config.rst
Comment (feorlen) [reply source/reference/minio-mc-admin/mc-admin-config.rst]: Also replaced other occurrences of `/home/events`

**Pull request merged**
